lib/puppet_litmus/serverspec.rb

```diff
@@ -229,10 +251,11 @@ def targeting_localhost?
   #
   # @param command [String] The puppet command causing the error.
   # @param result  [Array] The result struct containing the result
-  def report_puppet_apply_error(command, result)
+  def report_puppet_apply_error(command, result, acceptable_exit_codes)
     puppet_apply_error = <<-ERROR
 apply manifest failed
 `#{command}`
+with exit code #{result.first['result']['exit_code']} (expected: #{acceptable_exit_codes})
 ======Start output of failed Puppet run======
```

Comment:
this header and footer shoudl be amended. was it an error or an unexpected change ?